src/utils.rs

```diff
@@ -1,11 +1,35 @@
 // #[cfg(target_arch = "wasm32")]
 // pub use wasm_bindgen_rayon::init_thread_pool;
+use clap::Parser;
 
-#[cfg(target_arch = "wasm32")]
-use wasm_bindgen::prelude::*;
 #[cfg(target_arch = "wasm32")]
 use std::collections::HashMap;
+#[cfg(target_arch = "wasm32")]
+use wasm_bindgen::prelude::*;
 
+#[cfg(not(target_arch = "wasm32"))]
+#[derive(Debug, Parser)]
+#[command(about = "bevy gaussian splatting render pipeline plugin", version, long_about = None)]
+struct CliArgs {
```

Comment:
I think merging `CliArgs` and `GaussianSplattingViewer` structs would be great (can happen in a future PR though, let me know your bandwidth): https://github.com/mosure/bevy_gaussian_splatting/blob/0c28bac7f071c729282a400db3a6701b44ef3590/viewer/viewer.rs#L47

e.g. this allows args like `--height`, `--width`, `--hide_fps`, `--hide_editor`, etc.

Reply:
makes sense

Reply:
as for bandwidth, I plan on working / contributing on this project long term. I'll play around with merging the structs and see how it goes.